common/views/components/ResponsiveImage/ResponsiveImage.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import { useRef, useEffect } from 'react';
+
+import { convertImageUri } from '../../../utils/convert-image-uri';
+import { classNames } from '../../../utils/classnames';
+import { imageSizes } from '../../../utils/image-sizes';
+
+type Props = {
+  url: string,
+  width: number,
+  height: ?number,
+  sizesQueries: ?string,
+  alt: string,
+  extraClasses?: string,
+};
+
+const ResponsiveImage = ({
+  url,
+  width,
+  height,
+  sizesQueries,
+  alt,
+  extraClasses,
+}: Props) => {
+  const imgRef = useRef();
+
+  useEffect(() => {
```

Comment:
Do we want this to run as often as this, or only on load, which case we need to add the `[]` as it's deps?

Reply:
Because we change the `data-srcset` on pre-transformed lazyload elements, we have to [re-apply the lazyload class](https://github.com/aFarkas/lazysizes#attrchange--re-initialization-extension--strongly-recommended-if-you-use-react-angular-etc), otherwise the DOM doesn't update when the data changes. The alternative is to include the 'reinitialisation extension' mentioned, but it seemed like unnecessary overhead.